src/blueprints/timeline-state-resolver-types/viz-types.ts

```diff
@@ -0,0 +1,39 @@
+import { TimelineObject } from '../../model/entities/timeline-object'
+import { DeviceType } from '../../model/enums/device-type'
+
+export interface VizMseElementInternalTimelineObject extends TimelineObject {
+  content: {
+    deviceType: DeviceType.VIZMSE
+    type: VizType.ELEMENT_INTERNAL
+    templateName: string
+    templateData: string[]
+    showName: string
+  }
+}
+
+export interface VizMseLoadAllElementsTimelineObject extends TimelineObject {
+  content: {
+    deviceType: DeviceType.VIZMSE
+    type: VizType.LOAD_ALL_ELEMENTS
+  }
+}
+
+export interface VizMseContinueTimelineObject extends TimelineObject {
+  content: {
+    deviceType: DeviceType.VIZMSE
+    type: VizType.CONTINUE
+    direction: -1 | 1 | undefined // Taken directly from Blueprints. Perhaps can/should be changed.
```

Comment:
This is most likely the values that the physical Viz device expects, so we should keep that in mind if we change it.
Also, it's the values that TSR expects.

Reply:
Which means the comment `Perhaps can/should be changed` should be deleted.

Reply:
Part of comment is gone.